watsononlinestore/watson_online_store.py

```diff
@@ -43,10 +43,11 @@ def send_message(self, message):
 
         :param str message: The message to be sent to slack
         """
-        self.slack_client.api_call("chat.postMessage",
-                                   channel=self.channel,
-                                   text=message,
-                                   as_user=True)
+        for line in message.splitlines(True):
+            self.slack_client.api_call("chat.postMessage",
```

Comment:
This helps with the amazon results, but your are right. Need  to create a separate branch for that. I will remove...